src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+
+import '../../styles/transitiongroup.scss';
+
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  filteredTodo: Todo[];
+  removeTodo: (TodoId: number) => void;
+  selectedId: number[];
+  isAdding: boolean;
+  title: string;
+};
+
+export const TodoList: React.FC<Props> = ({
+  filteredTodo,
+  removeTodo,
+  selectedId,
+  isAdding,
+  title,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      <TransitionGroup>
+        {filteredTodo.map(todo => (
+          <CSSTransition
+            key={todo.id}
+            timeout={300}
+            classNames="item"
+          >
+            <TodoItem
+              todo={todo}
+              key={todo.id}
+              removeTodo={removeTodo}
+              selectedId={selectedId}
+              isAdding={isAdding}
+            />
+          </CSSTransition>
+        ))}
+
+        {isAdding && (
+          <CSSTransition
+            key={0}
+            timeout={300}
+            classNames="temp-item"
+          >
+            <TodoItem
+              key={Math.random()}
```

Comment:
Don't use `Math.random` here

https://stackoverflow.com/questions/29808636/when-giving-unique-keys-to-components-is-it-okay-to-use-math-random-for-gener

Reply:
![image](https://user-images.githubusercontent.com/79811705/200875293-ab244987-cd1e-43c7-9491-83b3bb7d5f67.png)
that was in task description btw 😄

Reply:
:harold: